backend/models/core/Session.js

```diff
@@ -1,59 +1,64 @@
-import DB, { fromAurora, AURORA_TYPE } from '@backend/data_access/DB';
-import UnauthorizedError from '@/components/errors/UnauthorizedError';
-import AppNotEnabledError from '@/components/errors/AppNotEnabledError';
-import InsufficientPermissionsError from '@/components/errors/InsufficientPermissionsError';
-import { log } from '@/lib/Logger';
-    
+import DB, { fromAurora, AURORA_TYPE } from "@backend/data_access/DB";
+import UnauthorizedError from "@/components/errors/UnauthorizedError";
+import AppNotEnabledError from "@/components/errors/AppNotEnabledError";
+import InsufficientPermissionsError from "@/components/errors/InsufficientPermissionsError";
+import { log } from "@/lib/Logger";
+
 class Session {
-  
-  static fields()  {
+  static fields() {
     return {
-      people_id: {auroraType: AURORA_TYPE.LONG, allowOnCreate: true, allowOnUpdate: true },
-      sme_tenant_id: {auroraType: AURORA_TYPE.LONG, allowOnCreate: true, allowOnUpdate: true},
+      people_id: {
+        auroraType: AURORA_TYPE.LONG,
+        allowOnCreate: true,
+        allowOnUpdate: true,
+      },
+      sme_tenant_id: {
+        auroraType: AURORA_TYPE.LONG,
+        allowOnCreate: true,
+        allowOnUpdate: true,
+      },
     };
-  };
+  }
 
-  static async getSession(session){ 
-    if(!session) throw new UnauthorizedError('Session not found.');
+  static async getSession(session) {
+    if (!session) throw new UnauthorizedError("Session not found.");
     const db = new DB();
-    const sql =  `SELECT people_id from sessions where session = :session AND session_expiry > NOW() LIMIT 1`;
+    const sql = `SELECT people_id from sessions where session = :session AND session_expiry > NOW() LIMIT 1`;
     const data = await db.executeStatement(sql, [
-      {name: 'session', value:{stringValue: session}},
+      { name: "session", value: { stringValue: session } },
     ]);
-    
-    if (data.records.length === 0) throw new UnauthorizedError('Session not found.');
-     
-    return data.records.map(([
-      {longValue: people_id}
-    ]) => ({
-      people_id
+
+    if (data.records.length === 0)
+      throw new UnauthorizedError("Session not found.");
+
+    return data.records.map(([{ longValue: people_id }]) => ({
+      people_id,
     }))[0];
   }
-  
+
   /**
    * assert({
    *  loggedIn: true,
    *  appsEnabled: ["trucking"],
    *  userHasPermission: ["manage"]
    * });
-   * 
+   *
    * The goal is to resolve permission handling in the least amount of Aurora Data API requests.
-   * 
+   *
    */
-  
+
```

Comment:
FYI: For context, this is inherited from SME (https://github.com/klaudsol/sme-amplify). SME is, codebase-wise, the ancestor of KlaudSol CMS. 

The way permissions are handed here does not make sense with KlaudSol CMS. We would be using [this in the near future](https://github.com/klaudsol/klaudsol-cms/issues/196)

Reply:
Thank you for bringing this to my attention, sir. I will certainly take it into careful consideration as I work on future pull requests related to this!